7_react/react1/week2/src/compoents/Counter.jsx

```diff
@@ -0,0 +1,19 @@
+import { useState, useEffect } from "react";
+
+function Counter() {
+  const [seconds, setSeconds] = useState(0);
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setSeconds((prevSeconds) => prevSeconds + 1);
+    }, 1000);
+
+    return () => clearInterval(interval);
```

Comment:
It's very good practice to remember to cleanup once the component is unmounted. So great job here 👍 👍